pallets/parachain-system/src/lib.rs

```diff
@@ -763,7 +765,10 @@ impl<T: Config> Pallet<T> {
 			weight_used += T::DmpMessageHandler::handle_dmp_messages(message_iter, max_weight);
 			<LastDmqMqcHead<T>>::put(&dmq_head);
 
-			Self::deposit_event(Event::DownwardMessagesProcessed(weight_used, dmq_head.head()));
+			Self::deposit_event(Event::DownwardMessagesProcessed {
+				used: weight_used,
+				dmq_head: dmq_head.head(),
```

Comment:
Is there a better event name than dmq_head? last_chain_head_processed? last_blockhash_processed?

Reply:
No, that is the correct name.